src/software/new_geom/rectangle.h

```diff
@@ -0,0 +1,112 @@
+#include "software/new_geom/convex_polygon.h"
+
+/**
+ * A rectangle.
```

Comment:
Could we indicate that this rectangle cannot be rotated?

Reply:
Fixed in latest update